src/components/ProfileHeader/ProfileHeader.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import { afterEach } from "vitest";
+import { ProfileHeader } from "./ProfileHeader";
+
+afterEach(() => {
+  cleanup();
+});
+
+describe("Header component for DEX CDC Portal", () => {
+  it("should render seven buttons with the correct text", () => {
```

Comment:
Maybe don't need to specify the exact number of buttons since that could change.  Can just say "some" buttons.